src/Sarif.Multitool/Rules/RuleResources.resx

```diff
@@ -140,10 +140,12 @@ URIs must conform to [RFC 3986](https://tools.ietf.org/html/rfc3986). In additio
 If the 'invocation' object specifies both 'startTimeUtc' and 'endTimeUtc', then 'endTimeUtc' must not precede 'startTimeUtc'. To allow for the possibility that the duration of the run is less than the resolution of the string representation of the time, the start time and the end time may be equal.</value>
   </data>
   <data name="SARIF2001_TerminateMessagesWithPeriod_Warning_Default_Text" xml:space="preserve">
-    <value>{0}: In rule '{1}', the message with id '{2}' does not end in a period. Write rule messages as complete sentences.</value>
+    <value>{0}: In rule '{1}', the message with id '{2}' does not end in a period. Express plain text rule messages as complete sentences. This guidance does not apply to Markdown messages, which might include formatting that makes the punctuation unnecessary.</value>
   </data>
   <data name="SARIF2001_TerminateMessagesWithPeriod_FullDescription_Text" xml:space="preserve">
-    <value>Write in complete sentences and end each sentence with a period. This guidance does not apply to Markdown messages, which might include formatting that makes the punctuation unnecessary.</value>
+    <value>The properties of a result's 'message' property must be consistent with the properties of the rule that the result refers to.
```

Comment:
Wrong message.

Reply:
thanks! yep, my copy pasta! fixing!


---
In reply to: [449246936](https://github.com/microsoft/sarif-sdk/pull/1960#discussion_r449246936) [](ancestors = 449246936)